### ISSUE TITLE
Replace react-json-view with CodeMirror for JSON body display

### ISSUE THREAD

**Issue by chriswhong** [qri-io/desktop#301]
With large and nested JSON bodies, the rendering of `body` chokes the app.

Let's replace `react-json-view` with a more static display using Codemirror (Codemirror is used on Qri Cloud to show JSON bodies, and is also a dependency of our markdown editor `react-simplemde-editor`

**Comment by chriswhong** [comment]
No longer relevant after #313 (used react-json-view options to show faster-rendering JSON for display)
